app/models/eligibility_verification.rb

```diff
@@ -9,11 +9,8 @@ class EligibilityVerification < ApplicationRecord
   validates :passenger, uniqueness: true
   validates :expiration_date, presence: true, if: :passenger_requires_validation
   validates :verifying_agency_id, presence: true, if: :passenger_requires_validation
-  validates :expiration_date, absence: { if: -> { verifying_agency.blank? },
-                                         message: 'cannot be entered without verifying agency' }
-  validates :expiration_date,
-            absence: { if: -> { passenger&.permanent? },
-                       message: 'may not be entered for permanent passengers.' }
+  validates :expiration_date, absence: { if: -> { verifying_agency.blank? }, message: :no_agency }
+  validates :expiration_date, absence: { if: -> { passenger&.permanent? }, message: :perm_pax }
```

Comment:
I was a little mad about this; I figured out this usage by just guessing it would work. The [Rails Guide](https://guides.rubyonrails.org/active_record_validations.html#message) says:

> The `:message` option accepts either a `String` or `Proc` as its value.

Reply:
Yeah I had to do validation messages a while ago and it took me way too long to figure out how to set it up